lib/tlIO/FFmpeg.cpp

```diff
@@ -177,16 +190,33 @@ namespace tl
             IPlugin::_init(
                 "FFmpeg",
                 {
-                    { ".mov", io::FileType::Movie },
+                    // Video Formats
+                    { ".avi", io::FileType::Movie },
```

Comment:
This should be a separate PR. Also as I have mentioned before, I do not want to add support for older/non-VFX related formats like webm and flv.

Reply:
I reverted to your list of formats only.